raiden/constants.py

```diff
@@ -75,11 +75,12 @@ class EthClient(Enum):
 
 TESTNET_GASPRICE_MULTIPLIER = 2.0
 
-GAS_REQUIRED_FOR_OPEN_CHANNEL = 111577
-GAS_REQUIRED_FOR_SET_TOTAL_DEPOSIT = 44882
+# as calculated on 0dbe840c366841b414a11c78d8721046440b2a15
```

Comment:
change to: calculated as of raiden-contracts@0dbe840c366841b414a11c78d8721046440b2a15
Also perhaps include a link to the commit

Reply:
Done